### ISSUE TITLE
Contact vs Patron name confusion?

### ISSUE THREAD

**Issue by anthonywhitford** [anthonywhitford/bic-lcf#3]
While working through a PoC implementation of LCF, it became aparent that "Patron" does not mean a Person in the real world. This was my expectation at first seeing that word. The "Contact" entity is actually a Person in the real world. 

I wonder whether this may lead to confusion within the library community, since i've already had conversations with Bibliotheca, who also seemed to have the understanding that Patron was a person?

**Comment by franciscave** [comment]
The reason for introducing the Contact entity – fairly late in the development of LCF v1.0 – was to address a requirement expressed by the Working Group that access to personal data such as names and addresses and languages should be kept separate from account-type data such as whether or not the Patron is blocked. Before this change such personal data elements were seen as part of the Patron entity. 

 

In the LCF v1.0 documentation the specification of E03 Patron includes a note that “Contact information is held in separate contact records for security and privacy reasons.”

 

LCF v1.0 does not require that every Patron entity has an associated Contact entity. This may be an error. See E03D02 Contact reference, which has cardinality ‘0-1’.

 

The key question here is: does the fact that personal data is held separately from other patron data mean that these have to be two logically separate entities, or can they be part of the same entity, with some internal system logic to determine when it is appropriate to return the whole record or the record minus personal data or the record minus non-personal data? At the time when LCF v1.0 was being finalised it seemed that the simplest thing was to create two separate entities, but we may need to re-visit that decision.

**Comment by anthonywhitford** [comment]
We intend to fold the definition of Contact down into Patron, leaving only the Patron entity, covering the account details and the personal details. The documentation and XSD will be updated shortly.

**Comment by anthonywhitford** [comment]
Revised the schema to reflect the changes, flattening the Contact entity down into Patron. Contact does still exist, and represents the method of contact for a Patron, as controlled by the CMT list. A contact now requires an ''identifier'' in order to be managed within the underlying database effectively, since you are also able to create new methods of contact for an existing Patron. For example when they advise of a new email address or phone number.

@franciscave Would you be able to reivew and provide feedback please?

**Comment by franciscave** [comment]
I have reviewed this change and I think it makes a lot of sense. Not least it makes my job of revising the documentation much easier!

**Comment by franciscave** [comment]
I've made changes to the LCF v1.0 specification, entity XML binding specification and code list CMT to implement the structural changes already drafted in the XSD. I've added one new code to CMT: '06' for 'Postal address'. All these changes need to be checked, especially E03 Patron and E09 Contact in the v1.0 specification and in the entity XML binding specification. In particular, am I flagging the changes in a sensible way?
